git-operator/chart/README.md

```diff
@@ -0,0 +1,42 @@
+
+keptn-gitops-operator
+===========
+
+Helm Chart for the keptn keptn-gitops-operator
+
+
+## Configuration
+
+The following table lists the configurable parameters of the keptn-gitops-operator chart and their default values.
+
+| Parameter                | Description             | Default        |
+| ------------------------ | ----------------------- | -------------- |
+| `apiUrl` | Keptn API service | `"http://api-gateway-nginx/api"` |
+| `watchNamespace` | Namespace that operator watches | `""` |
+| `helmservice.image.repository` | Container image name | `"docker.io/keptn/keptn-gitops-operator"` |
```

Comment:
For the keptn-sandbox repository the following should be the correct docker organization

```suggestion
| `helmservice.image.repository` | Container image name | `"docker.io/keptnsandbox/keptn-gitops-operator"` |
```